ibm/resource_ibm_database.go

```diff
@@ -1473,7 +1473,7 @@ func resourceIBMDatabaseInstanceUpdate(d *schema.ResourceData, meta interface{})
 		if err != nil {
 			return fmt.Errorf("Error updating database scaling group: %s", err)
```

Comment:
`"Error updating database scaling group: %s"` ==>` "Error updating database cpu auto scaling group: %s"`
Similar changes for` "auto_scaling.0.disk"`, `"auto_scaling.0.memory" `